src/App.js

```diff
@@ -10,7 +10,13 @@ let App = () => {
   let [color,setColor]=useState("")
   let [activated,setActivated]=useState("")
   return (
-      <div>
+    <div
+      style={{
+        backgroundColor: 'yellow',
+        color: 'lime',
+        fontFamily: 'Comic Sans MS'

```

Comment:
```suggestion 
        fontFamily: 'Arial'
```
        You should not use Comic Sans for anything on any of **my** webpages.